spec/features/routing/destinations/batch_update_spec.rb

```diff
@@ -183,5 +191,16 @@
         end.to have_enqueued_job(AsyncBatchUpdateJob).on_queue('batch_actions').with 'Routing::Destination', be_present, assign_params, be_present
       end
     end
+
+    context 'when routing tag ids field is empty' do
+      let(:assign_params) { { routing_tag_ids: '' } }
```

Comment:
value should be empty array

`let(:assign_params) { { routing_tag_ids: [] } }`